### ISSUE TITLE
Refresh on release

### ISSUE THREAD

**Issue by andrewda** [gitpoint/git-point#234]
Currently pages refresh when the page is pulled down. This doesn't feel natural — it should refresh on release instead.

**Comment by housseindjirdeh** [comment]
Are you referring to all pages that use `PullToRefresh` like the events screen/repository screen/org screen?

**Comment by andrewda** [comment]
Yes, as far as I know this happens on all screens.

**Comment by lex111** [comment]
What does "refresh on release" mean?

**Comment by FlorianTaut** [comment]
@lex111 i think @andrewda means when you release the screen so put away your finger the refresh action should start, right?

**Comment by andrewda** [comment]
Yea. Almost all apps I know of refresh when you let go, not when you pull up. Example:

![example](https://im.ezgif.com/tmp/ezgif-1-82a76aadc1.gif)

**Comment by lex111** [comment]
@andrewda thanks, now I understand what you, although it's unusual for me, or I'm just used to the fact that now :smile:

**Comment by housseindjirdeh** [comment]
So all pull to refresh functionality uses the [`RefreshControl`](https://facebook.github.io/react-native/docs/refreshcontrol.html) (either directly or through `FlatList` as a `onRefresh` prop). So on iOS - it works lovely in my opinion

![pulltorefresh](https://user-images.githubusercontent.com/12476932/29243211-ce42ffc4-7f4e-11e7-87b0-9f799a1c3285.gif)

However if I'm not mistaken, the last time I tried with Android it acted a little weirdly. It wasn't really `pulling` the list was it? It just shows the loading indicator on pull and that's it. @andrewda @lex111 are you noticing the same thing?

Attached an `android` tag to this ticket but let me know if you happened to also be referring to iOS @andrewda

**Comment by andrewda** [comment]
Wait @housseindjirdeh try on a profile or repository. That's where it acts strangely for me (on iOS).

**Comment by FlorianTaut** [comment]
Yeah when you pull down and don't leave you finger it refreshes the content and acts a bit strange
